FSharpWorkspaceShim/FSharpWorkspaceShim.fsproj

```diff
@@ -18,7 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="FSharp.Compiler.Service" />
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="3.2.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="3.4.0-beta2-19467-02" />
```

Comment:
Is there a central place you can put these version numbers, so they don't have to be updated in so many places?